### PR TITLE
fix(Scripts/Chat): Fix chat logger

### PR DESCRIPTION
**Pull request by Kitzunu**
## Changes Proposed:
This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [x] Scripts (bosses, spell scripts, creature scripts).
-  [ ] Database (SAI, creatures, etc).

If your pull request promotes complex changes that require a detailed explanation, please describe them in detail specifying what your solution is and what is it meant to address.

## Issues Addressed:

- Closes 

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [ ] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [ ] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: src/server/apps/worldserver/worldserver.conf.dist [hunk 1]
```diff
@@ -3949,6 +3949,7 @@ AuctionHouse.SearchTimeout = 1000
 
 Appender.Console=1,4,0,"1 9 3 6 5 8"
 Appender.Server=2,5,0,Server.log,w
+# Appender.Chat=2,4,15,chat_%s.log
```
Comment (Winfidonarleyan) [inline src/server/apps/worldserver/worldserver.conf.dist:3952]: why `%s` ?

Comment (Kitzunu) [reply src/server/apps/worldserver/worldserver.conf.dist:3952]: for date

File: src/server/scripts/World/chat_log.cpp [hunk 2]
```diff
@@ -36,82 +31,84 @@ class ChatLogScript : public PlayerScript
...
 
             case CHAT_MSG_EMOTE:
-                LOG_CHAT("emote", "Player {} emotes: {}",
+                LOG_INFO("chat.emote", "Player {} emotes: {}",
                     player->GetName(), msg);
                 break;
 
             case CHAT_MSG_YELL:
-                LOG_CHAT("yell", "Player {} yells (language {}): {}",
+                LOG_INFO("chat.yell", "Player {} yells(language{}) : {}",
```
[the remark this answers is not in the document]
Comment (Kitzunu) [reply src/server/scripts/World/chat_log.cpp:44]: Yea this change was accident

File: src/server/apps/worldserver/worldserver.conf.dist (hunk 1 above)
Comment (Kitzunu) [reply src/server/apps/worldserver/worldserver.conf.dist:3952]: and it will make a new file at every new worldserver start. Same as Appender.GM

**Review by Winfidonarleyan — APPROVED** [review]

**Pull request merged**